src/notAllUnique.js

```diff
@@ -0,0 +1,25 @@
+import { complement } from 'ramda';
+
+import allUnique from './allUnique';
+
+/**
+ * Returns true if at least one item of the list is repeated. `R.equals` is used to determine equality.
+ *
+ * @func notAllUnique
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.22.0|v2.22.0}
+ * @category List
+ * @sig [a] -> Boolean
+ * @param {Array} list The list of values
+ * @return {boolean}
+ * @see {@link https://ramdajs.com/docs/#equals|equals}
```

Comment:
`@see {@link RA.allEqual|allEqual}, {@link https://ramdajs.com/docs/#equals|equals}`

Reply:
I'll fix this myself after the merge.